Source/DTDLv2/RealEstateCore/Asset/Equipment/ICTEquipment/DataNetworkEquipment/EthernetSwitchPort.json

```diff
@@ -0,0 +1,57 @@
+{
+  "@id": "dtmi:org:w3id:rec:EthernetSwitchPort;1",
+  "@type": "Interface",
+  "description": {
+    "en": "Ethernet switch port."
+  },
+  "displayName": {
+    "en": "Ethernet Switch Port"
+  },
+  "contents": [
+    {
+      "@type": "Property",
+      "name": "poeType",
+      "displayName": {
+        "en": "PoE Type"
+      },
+      "schema": {
+        "@type": "Enum",
+        "valueSchema": "string",
+        "enumValues": [
+          {
+            "name": "Type1",
+            "displayName": "802.3af (802.3at Type 1), PoE",
+            "enumValue": "Type1"
+          },
+          {
+            "name": "Type2",
+            "displayName": "802.3at Type 2, PoE+",
+            "enumValue": "Type2"
+          },
+          {
+            "name": "Type3",
+            "displayName": "802.3bt Type 3, 4PPoE or PoE++",
+            "enumValue": "Type3"
+          },
+          {
+            "name": "Type4",
+            "displayName": "802.3bt Type 4, 4PPoE or PoE++",
+            "enumValue": "Type4"
+          }
+        ]
+      },
+      "writable": true
+    },
+    {
+      "@type": "Property",
+      "name": "portSpeed",
```

Comment:
Should this be a string or an enum field? Remember that we discussed it but don't recall the specifics. Are there too many possibilities for an enum?

Reply:
We did discuss an enum. Then when implementing I found this https://en.wikipedia.org/wiki/List_of_interface_bit_rates (see the LAN section) and decided against it.

Not strongly against it though. We can of course stick to the traditional Ethernet port speeds only, like 10BASE-X, 100BASE-X etc. or only enumerate the bandwith without mentioning (or implicitly referring to) the standard, eg 10Mbit/s, 100Mbit/s...

Can we consider briefly at upcoming TC?

Reply:
Suggestion: let's update this to use quantitative type extension from DTDLv3 preview.
E.g., the portspeed is a Property and a DataRate, with schema integer or float, and unit mebibitPerSecond or mebibytePerSecond

Reply:
https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/Docs/en-US/DTDL.quantitativeTypes.v1.md

Reply:
Changed with recent commit.